scss/statics/top/unlogined.static.scss

```
@@ -106,7 +106,7 @@ h6:not(.overwrite) {
 
 .text-foreground {
     $tw-text-opacity: 1;
-    color: hsl(var(--foreground-default / $tw-text-opacity));
+    color: hsl(var(--brand-default) / $tw-text-opacity);
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)